JetSet/JetSet/MenuView.swift

```diff
@@ -9,88 +9,85 @@ import SwiftUI
 
 struct MenuView: View {
     var body: some View {
-        
-        NavigationView {
```

Comment:
It looks like a lot of changes but it's just a matter of removing the NavigationView from the hierarchy. This causes every line to change its indentation which is annoying.

Reply:
Basically, since `TitleView` uses a NavigationStack to present `MenuView`, the `MenuView` itself is already going to be within the `NavigationStack` context. `NavigationView` is kind of like an older version of `NavigationStack`, so if `MenuView` remained with this definition, you would have had nested `NavigationStack`s which isn't what we want in this context.